packages/shared/lib/utils.ts

```diff
@@ -155,7 +155,16 @@ export const isValidHttpsUrl = (url) => {
  * @returns True if it matches the bech32 format.
```

Comment:
I just realized this `@returns` is wrong, maybe we can rephrase it to something like `@returns error message string if it doesnt match the bench32 format`?

Reply:
Updated